cli/config/configuration.toml.in

```diff
@@ -134,6 +134,12 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # used for 9p packet payload.
 #msize_9p = @DEFMSIZE9P@
 
+# If true and vsocks are supported, use vsocks to communicate directly
+# with the agent and no proxy is started, otherwise use unix
+# sockets and start a proxy to communicate with the agent.
+# Default false
+use_vsock = false
```

Comment:
To be convenient for users, you can make this `# use_vsock = true` so that it can be enabled simply by removing the leading `#`. We've done it for other disabled features too.